### PR TITLE
Created components

### PR DESCRIPTION
**Pull request by thecodechaser**
##  This branch includes the following features:
-  Created calculator component.
-  Exported the calculator component to the main component(App.js).
-  Created markup for the calculator.
-  Applied styling for the calculator.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,6 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => <Calculator />;
```
Comment (bimbolabuari) [inline src/App.js]: As required, Let's refactor this to a class-based component ✊

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @thecodechaser, 

You have done a good job so far 🚀  

## Highlights 🎉 

-  Project has a directory called components.
- `Calculator.js` file is inside the components directory.
- Calculator.js component is class-based.

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: src/App.js (hunk 1 above)
Comment (thecodechaser) [reply src/App.js]: > As required, Let's refactor this to a class-based component fist_raised

Hello @bimbolabuari 
Thank you for your review
I have tried with class-based component for App
But it was giving me linter errors, That it should be pure function and i had to change it again to function-based component
Can you please help me with that ?

**Review by AshakaE — APPROVED** [review]
### Hi @thecodechaser   🙋‍♀️, 

# STATUS APPROVED ✔
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**